src/common/__tests__/cluster-store.test.ts

```diff
@@ -94,6 +95,7 @@ describe("cluster-store", () => {
     mainDi.permitSideEffects(getConfigurationFileModelInjectable);
     mainDi.permitSideEffects(appVersionInjectable);
     mainDi.permitSideEffects(clusterStoreInjectable);
+    mainDi.permitSideEffects(fsInjectable);
```

Comment:
Doesn't seem right, but neither does `mainDi.permitSideEffects(getConfigurationFileModelInjectable);`. Guess there is good reason for it for now :)

Reply:
Well until we factor out all of this (and probably the library that we are using) we have to continue to use `mockFs`.